src/components/TagDescriptionEditor.js

```diff
@@ -43,7 +75,9 @@ export default class TagDescriptionEditor extends React.Component {
           </div>
         </div>)}
         <div className='footer'>
-          <button onClick={() => onSave(tags)} className='ok'>Create</button>
+          {creating
+          ? <button onClick={createTag} className='ok'>Create</button>
```

Comment:
actually in this case the only difference in the two values is the onClick handler so you might as well shrink the conditional expression to change just that